### DISCUSSION TITLE
ValueError with bt.plot()

### DISCUSSION THREAD

**Discussion by aliencaocao** [kernc/backtesting.py#368] [Q&A]
```
BokehDeprecationWarning: HSL() was deprecated in Bokeh 2.3.1 and will be removed, use RGB() or named colors instead.
BokehDeprecationWarning: HSL() was deprecated in Bokeh 2.3.1 and will be removed, use RGB() or named colors instead.
Traceback (most recent call last):
  File "C:/Users/alien/Documents/PyCharm Projects/TDA Quant/knn.py", line 126, in <module>
    backtest.plot()
  File "C:\Program Files\Python38\lib\site-packages\backtesting\backtesting.py", line 1704, in plot
    plot(
  File "C:\Program Files\Python38\lib\site-packages\backtesting\_plotting.py", line 591, in plot
    _plot_superimposed_ohlc()
  File "C:\Program Files\Python38\lib\site-packages\backtesting\_plotting.py", line 445, in _plot_superimposed_ohlc
    raise ValueError('Invalid value for `superimpose`: Upsampling not supported.')
ValueError: Invalid value for `superimpose`: Upsampling not supported.
```
my backtest code:
```py
backtest = Backtest(data, Strat, cash=2500, commission=0.0, margin=1/1, trade_on_close=False, hedging=False, exclusive_orders=False)
```
My input data is a dataframe with datetime as index.

**Comment by kernc** [comment]
Have you, as is customary in development, considered first trying to [use the search](https://www.google.com/search?q=Invalid+value+for+`superimpose`%3A+Upsampling+not+supported.)?

Issue https://github.com/kernc/backtesting.py/issues/233.

**Reply by aliencaocao** [reply]
Of course I did, but just within this forum since there was specific instructions for us to ask questions in the forum instead of opening issues. Searching 'ValueError: Invalid value for `superimpose`: Upsampling not supported.' will give no result, therefore I opened a thread here

**Reply by kernc** [reply]
Indeed. There's also the issue tracker, but the above query would equally yield zero results prior to the bug renaming. Thanks. :sweat_smile:
